libs/feature/editor/src/lib/components/record-form/record-form.component.html

```diff
@@ -33,7 +33,15 @@
               trackBy: fieldTracker
             "
           >
-            <ng-container *ngIf="!field.config.hidden">
+            <gn-ui-check-toggle
```

Comment:
This render logic should be in a new component (e.g. `form-field-open-data.component`).
The new component can be rendered inside of /components/record-form/form-field/form-field.component.html and the model gets checked in the corresponding `form-field.component.ts` file. You can take the example from the title:
`  get isTitle() {
    return this.model === 'title'
  }`